src/testing/TestContext.cpp

```diff
@@ -213,6 +219,17 @@ void TestContext::initializePetsc()
   }
 }
 
+void TestContext::initializeGinkgo()
+{
+  if (!invalid && _kokkos) {
```

Comment:
Kokkos or Ginkgo? :sweat_smile:

Reply:
I think this is mainly about initializing the device. Hence, it doesn't really matter here.

Reply:
The names should be consistent tough. Requirement, option, function